src/styles/components/_sideBarContainer.scss

```diff
@@ -0,0 +1,46 @@
+.target-icon{
+  margin-top: 10px;
+  width: 30px;
+	height: 30px;
+}
+
+.smilies-img-sidebar{
+  margin-top: 10px;
+  width: 48px;
+	height: 28px;
+}
+
+.sidebar{
+  width: 318px;
+	height: inherit;
+	text-align: center;
+  background-color: #ffffff;
+}
+
+.sidebarHeader{
+	width: 100%;
+	height: 50px;
+	display: inline-block;
+	background-color: #2fbcf7;
+}
+
+.sidebarHeaderTitle{
+  font-family: sans-serif;
+	font-size: 20px;
+	font-weight: 500;
+	letter-spacing: 3px;
+  color: #ffffff;
+}
+
+.sidebarSubHeader{
+	width: 100%;
+  margin-top: 6px;
+  margin-bottom: 0px;
```

Comment:
use margin shorthand: `margin: top right bottom left`
Also, don't use units for the zero value